src/libs/collect.c

```diff
@@ -2169,6 +2204,8 @@ void gui_init(dt_lib_module_t *self)
   GtkBox *box;
   GtkWidget *w;
 
+  const gboolean show_module = dt_conf_get_bool("plugins/lighttable/collect/module");
```

Comment:
Probably time to remove this now ? This was for debug purpose, right ? Applying your PR I was surprised to not find the two new filters !

Reply:
My idea was not to bother people who are not concerned with migration with this.
But of course we can show them all the time. not problem.

Reply:
> Activate those filter by default.

Ok, I do this.